node/src/chains/common.rs

```diff
@@ -109,6 +107,8 @@ pub fn runtime_genesis_config(
 			"validatorCount": validator_count,
 			"minimumValidatorCount": 1,
 			"stakers": stakers,
+			"minNominatorBond": constants::staking::MIN_NOMINATOR_BOND,
+    		"minValidatorBond": constants::staking::MIN_VALIDATOR_BOND,
```

Comment:
nit: indentation must be causing the lint failure

Reply:
done :)